hibernate-core/src/main/java/org/hibernate/tuple/NonIdentifierAttribute.java

```diff
@@ -24,8 +24,6 @@
 
 	public boolean isNullable();
 
```

Comment:
I'm not sure if changing this interface could cause problems. It would probably be safer to deprecate it instead.